docs/help/contributor/style-guide/style-and-formatting.md

```diff
@@ -60,6 +60,10 @@ between the number and the unit.
 |Click **Fork** | Click "Fork"
 |Select **Other**      | Select "Other"
 
+### Use tables for definition lists
+
+When listing terms and their definitions, use table formatting instead of definition list formatting.
```

Comment:
I see that all guidelines surrounding this one are providing a good/bad example instead of a sentence. Should we remain consistent and use an example instead/additionally here too?

Reply:
Fair -- it was a little awkward to add the example, but not as bad as I thought.